discord/README.md

```diff
@@ -1,51 +1,47 @@
-# Bobby B Bot - Discord version
-[![Build Status](https://travis-ci.org/bobby-b-bot/discord.svg?branch=master)](https://travis-ci.org/bobby-b-bot/discord) ![GitHub release](https://img.shields.io/github/release/bobby-b-bot/discord.svg) ![GitHub All Releases](https://img.shields.io/github/downloads/bobby-b-bot/discord/total.svg) ![GitHub issues](https://img.shields.io/github/issues-raw/bobby-b-bot/discord.svg) ![GitHub](https://img.shields.io/github/license/bobby-b-bot/discord.svg) [![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/L3L814HD5)
+# Trigger Happy Bot
 
-In this repository you can find the Discord version of the Bobby B Bot.  
+**TODO: Agree on good bot name now that BobbyB has been abstracted**
```

Comment:
I'm good with Trigger Happy Bot for working title, we can change the repo name if you want.